### PR TITLE
Added ability to configure the Redis connection.

### PR DESCRIPTION
**Pull request by forest**
* Redis setter code shamefully borrowed from the Split gem.
   https://github.com/andrew/split/blob/master/lib/split.rb#L32
 * Added Redis configuration docs to the read me.
 * Added Redis::Namespace and a default forgetsy namespace.

### CODE REVIEW

**Comment by deevis** [comment]
Well, this is kinda a big deal.  I mean, the gem is cool and thank you for making it available to us, but assuming Redis is running on localhost and on the default port aint gonna fly for us.

I'm curious why this pull request hasn't been incorporated into the project?

@forest, looks like we're going to piggy-back on your fork until this is incorporated - thanks!

**Comment by kris** [comment]
Just setup your `redis.yml` and configure it how you want? Perhaps I'm missing something ^_^

File: spec/delta_spec.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@
     it 'passes options on to sets' do
       opts = { decay: false }
       mock_set = double()
-      mock_set.should_receive(:fetch).with(opts) { [] }
+      expect(mock_set).to receive(:fetch).with(opts) { [] }
```
Comment (cavvia) [inline spec/delta_spec.rb]: this syntax isn't supported in `rspec 2.13.0`, though the test itself passes with the old syntax. will clean up on master.

**Comment by cavvia** [comment]
Apologies. I never received a notification for this and I've been on a sabbatical. Clearly a yaml file is an insufficient form of run time configuration for a standalone gem, so thanks for this. Merging in.

**Pull request merged**